src/pyramid/interfaces.py

```diff
@@ -482,8 +482,40 @@ def __call__(self, **kw):
         """
 
 
+class ISecurityPolicy(Interface):
+    def identify(request):
+        """ Return an object identifying a trusted and verified user. """
+
+    def permits(context, request, identity, permission):
```

Comment:
I think for consistency I'd reverse the signature and put request before context.